### PR TITLE
make infer test more stable.

### PR DESCRIPTION
**Pull request by panyx0718**

### CODE REVIEW

File: paddle/contrib/inference/test_paddle_inference_api_impl.cc [hunk 1]
```diff
@@ -144,8 +144,8 @@ TEST(paddle_inference_api_impl, image_classification) {
   float* data = static_cast<float*>(outputs[0].data.data);
   float* lod_data = output1.data<float>();
   for (size_t j = 0; j < len / sizeof(float); ++j) {
-    EXPECT_LT(lod_data[j] - data[j], 1e-10);
-    EXPECT_GT(lod_data[j] - data[j], -1e-10);
+    EXPECT_LT(lod_data[j] - data[j], 1e-3);
+    EXPECT_GT(lod_data[j] - data[j], -1e-3);
```
Comment (luotao1) [inline paddle/contrib/inference/test_paddle_inference_api_impl.cc]: e-3会太大了么，e-6或者e-7比较好一点。

Comment (Superjomn) [reply paddle/contrib/inference/test_paddle_inference_api_impl.cc]: @panyx0718 这里为啥 1e-10, 1e-3 两种精度？可以统一成上面luotao提的吗

Comment (panyx0718) [reply paddle/contrib/inference/test_paddle_inference_api_impl.cc]: Done

Comment (panyx0718) [reply paddle/contrib/inference/test_paddle_inference_api_impl.cc]: I looked at some failed tests. some diff is about 1e-5. I made it 1e-3 to be stable.

Comment (luotao1) [reply paddle/contrib/inference/test_paddle_inference_api_impl.cc]: 请问为什么会有这么大的差距呢？

**Pull request merged**